frontend/src/components/commons/PopupBox.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { Component } from "react";
+import { Modal } from "react-bootstrap";
+import PropTypes from "prop-types";
+
+export const BUTTON_TYPE = {
+  primary: "btn btn-primary",
+  secondary: "btn btn-secondary"
+};
+
+const styles = {
+  boxshape: {
+    borderWidth: "1px 1px 0 1px",
+    borderStyle: "solid",
+    borderColor: "#00565e",
+    borderRadius: "5px 5px 5px 5px"
+  },
+  boxContent: {
+    paddingLeft: 20,
+    paddingRight: 20
+  },
+  modalHeader: {
+    display: "table",
+    width: "100%"
+  },
+  modelTitle: {
+    position: "absolute"
+  },
+  buttonsZone: {
+    width: "100%"
+  },
+  leftBtnLocation: {
+    float: "left"
+  },
+  rightBtnLocation: {
+    float: "right"
+  },
+  buttonSize: {
+    minWidth: 125
+  }
+};
+
+class PopupBox extends Component {
+  constructor(props, context) {
+    super(props, context);
+
+    this.state = {
+      show: false
+    };
+
+    this.PropTypes = {
+      title: PropTypes.string.isRequired,
+      description: PropTypes.string.isRequired,
+      leftButtonType: PropTypes.string.isRequired,
+      leftButtonTitle: PropTypes.string.isRequired,
+      leftButtonAction: PropTypes.func,
+      rightButtonType: PropTypes.string.isRequired,
+      rightButtonTitle: PropTypes.string,
+      rightButtonAction: PropTypes.func
+    };
+  }
+
+  handleClose = () => {
+    this.setState({ show: false });
+  };
+
+  handleShow = () => {
+    this.setState({ show: true });
+  };
+
+  buttonOneCloseAndAction = () => {
+    if (this.props.leftButtonAction) {
+      this.props.leftButtonAction();
+    }
+    this.setState({ show: false });
+  };
+
+  buttonTwoCloseAndAction = () => {
+    if (this.props.rightButtonAction) {
+      this.props.rightButtonAction();
+    }
+    this.setState({ show: false });
+  };
+
+  render() {
+    const {
+      title,
+      description,
+      leftButtonType,
+      leftButtonTitle,
+      rightButtonType,
+      rightButtonTitle
+    } = this.props;
+
+    return (
+      <div>
+        <button variant="primary" onClick={this.handleShow}>
```

Comment:
Instead of making include a button to open it, can it just be a dialog?
Right now this has demo code that won't be used and I don't think it should be included.

Reply:
Good catch! Indeed, this was just for testing. I removed it.
See commit #https://github.com/code-for-canada/project-thundercat/pull/126/commits/5eb8916eba0548e4017ff7180286c75f8b4ec41d